### ISSUE TITLE
Request: disable "OFF" function

### ISSUE THREAD

**Issue by mw75** [jneilliii/OctoPrint-Tasmota#209]
Hi there!

Would like to be able to disable the manual power down on the icons in the top bar.

I use one PI running all the time to turn on my other printers with there corresponding "pi's" (Thinclients in my case). These systems shutdown after print and turn off there socket delayed while shutdown. So to protect myself for an unwanted power los with heated nozzel and mounted filesystems i would like to disable "OFF" function completely or at least attach a confirmation dialog.

Thanks and regards,
Mario Wolff

**Comment by jneilliii** [comment]
If you don't enable idle timeout then they won't turn off on their own, only when they're told to by pressing a button. However, you seem to be concerned with powering off under certain conditions. In those cases it's best to use the idle timeout with the wait for temperatures settings. It might help me understand better what you're goal is if I could see your settings you mentioned.

**Comment by mw75** [comment]
You misunderstood my use case. I have one(!) system controlling which controls 4(!) Tasmota outlets. This system and it's printer is not connected to a Tasmota outlet at all!

I just use the octoprint Web-UI to remotely turn ON(!) the printer and it's thin-client i want to use. After this usage the thin-client automatically shuts down and also turns OFF(!) his own Tasmota outlet.

As the buttons are quite near to each other, it already happend, that i turned OFF printer three (for example) when intended to turn ON printer two (also example). 

With this usage i never ever turn OFF an outlet by my self! I would rather shut down the corresponding thin-client and it would turn OFF his outlet by itself.

That all already works! There is no problem a long as i misplaced a mouse click and that's all i would like to avoid!

Your great plugin supports multiple outlets on multiple Tasmota devices. What is the intended use for this?

**Comment by jneilliii** [comment]
> Your great plugin supports multiple outlets on multiple Tasmota devices. What is the intended use for this?

the idea was to be able to control power to printer, pi and other peripheral devices, like led lights for example.

I suppose in your use case the OctoPrint that you are controlling these other devices from may not always be in a printing state? The option in the individual device's settings `Warn While Printing` will actually prompt prior to performing the off function.

**Comment by mw75** [comment]
Right! The controlling Pi is embedded in an Ender3 and controls this printer internally with PSU.GPIO. The other Thin-Client-Printer combos are completely independent. So `Warn While Printing` does not fit my needs.

**Comment by jneilliii** [comment]
Got it. I don't see why I couldn't change the logic of that warn while printing option to a show confirmation always on the UI side. I'll review when I have some time available to see how much that will impact the code.
